conf/messages.en

```diff
@@ -350,6 +349,10 @@ form.import-questions.hasALVS.no=No
 form.import-questions.freightType.Maritime=Maritime
 form.import-questions.freightType.Air=Air
 form.import-questions.freightType.RORO=Rail, road, or roll-on, roll-off (RORO)
+form.import-questions.vessel-details.vesselName=Vessel name
```

Comment:
These are largely fine but could you amend the error messages (at present lines 322 & 323)

error.timeOfArrival.hour.invalid-digits=Time of arrival must only contain numbers
error.timeOfArrival.minutes.invalid-digits=Time of arrival must only contain numbers

<img width="477" alt="image" src="https://user-images.githubusercontent.com/55190161/96859933-06492080-145a-11eb-981d-dbb4324009de.png">